src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -234,12 +206,17 @@ class DrawCall {
       size -= 1; // Shave off one byte from the buffer size.
       length -= 1; // And one whole element from the writeBuffer.
     }
-    return this.createBufferWithContent(GPUBufferUsage.VERTEX, size, 0, vertexArray, 0, length);
+    const buffer = this.test.device.createBuffer({
+      size,
+      usage: GPUBufferUsage.VERTEX | GPUBufferUsage.COPY_DST, // Ensure that buffer can be used by writeBuffer
+    });
+    this.test.device.queue.writeBuffer(buffer, 0, vertexArray.slice(0, length));
+    return buffer;
   }
 
   // Create an index buffer from |indexArray|
   private generateIndexBuffer(indexArray: Uint32Array): GPUBuffer {
-    return this.createBufferWithContent(GPUBufferUsage.INDEX, indexArray.byteLength, 0, indexArray);
+    return this.test.makeBufferWithContents(indexArray, GPUBufferUsage.INDEX);
```

Comment:
nit: this is so simple now that imho we could inline it in the callers (it would make it clear we're just uploading the data)

Reply:
Done.